triggerReleasePipeline.sh

```diff
@@ -43,27 +43,28 @@ checkArgs $#
 # cleanup properties if exists from previous run
 git clean -fd # same as rm -rf ${SCRIPT_DIR}/properties but wont touch ${WORKSPACE}/tracking file
 
-# for first time if does not have repo locally yet
-cloneGitHubRepo $ADOPTIUM_REPO
```

Comment:
for the first time we need to get a local bare repo and set the correct remote origin?

Reply:
ah, i see, you have the ""{ADOPTIUM_REPO}"